### ISSUE TITLE
System.InvalidOperationException: 

### ISSUE THREAD

**Issue by anson1124** [iiwaasnet/kino#10]
System.InvalidOperationException: 请求的性能计数器不是自定义计数器，它必须初始化为 ReadOnly。
   在 System.Diagnostics.PerformanceCounter.InitializeImpl()
   在 System.Diagnostics.PerformanceCounter..ctor(String categoryName, String counterName, String instanceName, Boolean readOnly)
   在 kino.Core.Diagnostics.Performance.SafePerformanceCounter.CreatePerfCounter() 位置 C:\Users\Administrator\Downloads\kino-master\kino-master\src\kino.Core\Diagnostics\Performance\SafePerformanceCounter.cs:行号 32

**Comment by iiwaasnet** [comment]
This exception means, that perfcounters are not installed. It doesn't affect in anyway further functioning, hence they will be just "silently" off (as class name assumes, they are Safe :) ). You can check [here](https://github.com/iiwaasnet/kino/blob/master/src/kino.Rendezvous.Service/ServiceHost.cs) one of the possible ways to have perfcounters installed:
**_installutil_ kino.Rendezvous.Service.exe**
Full supported syntax [here](https://github.com/iiwaasnet/WindowsServiceHost).
Basically, this is about triggering the [PerformanceCounterInstaller](https://github.com/iiwaasnet/kino/blob/master/src/kino.Core/Diagnostics/Performance/PerformanceCounterInstaller.cs)

**Comment by anson1124** [comment]
thanks
